experiments.json

```diff
@@ -238,5 +238,20 @@
       "threshold": 0.2,
       "description": "Edit Top Sites is enabled"
     }
+  },
+  "pocketStories": {
+    "name": "Trending stories provided by Pocket",
+    "active": false,
```

Comment:
This should be `true` so it actually shows up in our list of experiments – if you want to ship it off for users, just set the `threshold` to `0`